src/main/fc/fc_msp.c

```diff
@@ -822,6 +823,20 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFn
         }
         break;
 
+#ifdef VTX_SMARTAUDIO
```

Comment:
Well, according to the src/main/target/common_post.h RTC6705 and SMARTAUDIO are mutially exclusive. The VTX API will be soon rewritten to add a generic interface for all kind of VTX's like Unify, Trump or RTC6705. For the time being it would be just nice to access the SmartAudio functionality not only from an OSD, but also via MSP.